.github/workflows/build_nilrt.yml

```diff
@@ -25,6 +25,13 @@ jobs:
       with:
         access_token: ${{ github.token }}
 
+    # If 2 large cpp files are compiled in parallel, we can run out of memory.
```

Comment:
TODO: This change tickled the build so that the nilrt build is running out of memory. This is something we've been pushing up against since I enabled `-j2`. It just depends on which objects get built in parallel. I'm going to try out this fix, which has worked in other branches and then re-ping owners on the proposal.

Reply:
I'm going to remove `-j2` instead of doing the more swap option. Running with `-j2` and more swap takes the same amount of time. Discussed with @SparkingSpork